ducktape/services/service.py

```diff
@@ -76,7 +76,9 @@ def __init__(self, context, num_nodes=None, cluster_spec=None, *args, **kwargs):
         self.context = context
 
         self.nodes = []
-        self.allocate_nodes()
+        self.skip_nodes_allocation = kwargs.get("skip_nodes_allocation", False)
```

Comment:
how does this kwarg get passed, if there's no mechanism to bring the command line argument into this function?

Reply:
Type in parser. Fixed. Thank you!